otwcode/otwarchive
# AO3-6004 show hidden bookmarks on a bookmarkable to admins

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6004

## Purpose

Allows admins to show hidden bookmarks on a bookmarkable

## Testing Instructions

 - Go to a bookmark index page with bookmarks
 - Hide a bookmark
 - When page refreshes, see "Make Bookmark Visible"
 - Clicking "Make Bookmark Visible" should refresh page and bring back bookmark

## Credit

Brandon W (he/him/they/them)
